voice-search/voice-search-impl/src/main/java/com/duckduckgo/voice/impl/VoiceSearchActivityLauncher.kt

```diff
@@ -130,9 +130,10 @@ class RealVoiceSearchActivityLauncher @Inject constructor(
         activity.window?.decorView?.rootView?.let {
             blurRenderer.addBlur(it)
         }
+        val stringSource = if (this::_source.isInitialized) _source.paramValueName else ""
```

Comment:
I honestly think we should just **not** make this variable `lateinit` anymore. We should just make it Nullable.

Reply:
I don’t remember much why I set this to lateinit before, but looking now, nothing is enforcing the order of the method calls, so we shouldn’t use lateinit